content/github/customizing-your-github-workflow/github-extensions-and-integrations.md

```diff
@@ -30,8 +30,12 @@ With the {% data variables.product.prodname_dotcom %} for Visual Studio Code ext
 
 ### Project management tools
 
-To streamline your workflow, you can integrate your {% data variables.product.product_name %} personal account or organization with third-party project management tools, such as Jira.
+You can integrate your {% data variables.product.product_name %} personal account or organization with third-party project management tools, such as Jira, slack.
 
 #### Jira Cloud and {% data variables.product.product_name %}.com integration
 
 You can integrate Jira Cloud with your personal or organization account to scan commits and pull requests, creating relevant metadata and hyperlinks in any mentioned Jira issues.
+
+#### Slack and {% data variables.product.product_name %} integration
+
+You can integrate Slack with your personal or organization account to subscribe for notifications, close or open issues, refer issues and pull requests without leaving Slack.
```

Comment:
I think it would be good to make it clearer what "refer issues and pull requests" means. How about:

```suggestion
You can integrate Slack with your personal or organization account to subscribe for notifications, close or open issues, and provide rich references to issues and pull requests without leaving Slack.
```